docs/development/threat_model.rst

```diff
@@ -156,18 +156,18 @@ What the Workstations can achieve
 
 -  The **Admin Workstation** requires Tails with a persistent volume,
    which stores information such as GPG and SSH keys, as well as a
-   `database with passphrases </docs/install.md#passphrase-database>`__
+   `database with passphrases </docs/passphrases.rst>`__
```

Comment:
I'm not sure whether this link should be pointing to `passphrases.rst` (which is just a high-level overview of which passphrases are needed for each user role in SecureDrop) or the section about KeePassX passphrase databases in the Tails Guide.